recipes/openfisca-core/meta.yaml

```diff
@@ -0,0 +1,87 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "OpenFisca-Core" %}
+{% set version = "35.7.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  sha256: 2e199a2a504625393f17dd26c363d3af1eda4f78a0470e40a87b2d9684bb0724
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - openfisca = openfisca_core.scripts.openfisca_command:main
+    - openfisca-run-test = openfisca_core.scripts.openfisca_command:main
+
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6,<4.0
+    - dpath >=1.5.0,<3.0.0
+    - nptyping ==1.4.4
+    - numexpr >=2.7.0,<=3.0
+    - numpy >=1.11,<1.21
+    - psutil >=5.4.7,<6.0.0
+    - PyYAML >=3.10
+    - sortedcontainers ==2.2.2
```

Comment:
do you really need this specific version of `sortedcontainers` and the other dependencies?

Reply:
yes, that's the case also in PyPi for this project, as it is related to state tax computation, we want to be sure that upgrading deps are carefully checked.